pkg/sql/codegen/pai/codegen.go

```diff
@@ -18,34 +18,76 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"regexp"
+	"strings"
 	"text/template"
 
+	"sqlflow.org/gomaxcompute"
+
 	"sqlflow.org/sqlflow/pkg/sql/codegen/tensorflow"
 	"sqlflow.org/sqlflow/pkg/sql/ir"
 )
 
 const entryFile = "entry.py"
 
+func getTableFromSelect(dataSource, trainSelect string) (string, string, error) {
+	// FIXME(typhoonzero): copied from tensorflow/codegen.go, should remove this and use the temp table
+	// in the workflow
+	fromRegex, err := regexp.Compile("FROM[\\s\\n]+([\\w\\.]*)")
+	if err != nil {
+		return "", "", err
+	}
+	matches := fromRegex.FindAllStringSubmatch(trainSelect, -1)
+	if len(matches) != 1 {
+		return "", "", fmt.Errorf("only support simple SQL query, but got %s", trainSelect)
+	}
+	paiTableFull := matches[0][1]
+	paiDatabase := ""
+	paiTable := ""
```

Comment:
How about renaming these variables with an `odps` prefix or without any prefix? After all, we are in the `pai` package,a `pai` prefix is redundant.